servers/rendering/renderer_rd/renderer_canvas_render_rd.cpp

```diff
@@ -1643,7 +1689,10 @@ RendererCanvasRenderRD::RendererCanvasRenderRD() {
 			global_defines += "#define MAX_LIGHTS " + itos(DEFAULT_MAX_LIGHTS_PER_RENDER) + "\n";
 		}
 
-		global_defines += "\n#define SAMPLERS_BINDING_FIRST_INDEX " + itos(SAMPLERS_BINDING_FIRST_INDEX) + "\n";
+		global_defines += "#define BATCH_MAX_TEXTURES " + itos(batch_available_texture_slots) + "\n";
+		if (push_pad_size > 0) {
+			global_defines += "#define PUSH_PAD_SIZE " + itos(push_pad_size) + "\n";
+		}
```

Comment:
If the push constant is not a multiple of 16, we need to add padding, and that is what this is for.